integrations/extensions/starter-kits/ibm-rpa/README.md

```diff
@@ -0,0 +1,107 @@
+# IBM RPA Custom Extension
+
+The IBM Robotic Process Automation (RPA) offering helps automate business and IT processes at scale with the ease and speed of traditional RPA. Software robots, or bots, can act on AI insights to complete tasks with no lag time and enable you to achieve digital transformation.
+
+This extension starter kits shows how to integrate IBM RPA into a conversational flow built with Watson Assistant. The integration relies on two cloud functions included with this starter kit that sit between Watson Assistant and IBM RPA. These cloud functions handle things like IBM RPA authentication and conversion of the REST API calls defined in the OpenAPI spec imported into Watson Assistant (also included in the start kit) to APIs supported by IBM RPA.
+
+One important caveate about this starter kit is that it does not contain the backend RPA distribution needed to complete an end-to-end demo. The purpose of the starter kit is to provide template assets needed to start building the integration but practitioners may need help from the IBM RPA client engineering team to build a working demo. IBM employees can put in a request for help through the Customer Care Jumpstart program. Please check the [intake process here through ISC](https://ibm.box.com/s/m715465sjeai35dz58rdp72t35z4z58m).
+
+## Starter Kit Overview
+
+Here is a list of the components included with this starter kit along with details on the purpose of each file:
+
+| File Name | Details |
+|-----------|---------|
+|cfInsert.js| Cloud function template used to insert an RPA item (e.g. job) into a queue to be executed. |
+|cfGetStatus.js| Cloud function template used to query RPA for item status. This is needed because RPA jobs can take longer than 30 seconds to complete which means you most poll this cloud function for the jobs completion status. |
+|RPAAsynchExtensionAPISpec.json| OpenAPI spec template that needs to be imported into Watson Assistant in order to integrate with the two cloud functions.|
+|RPAWA Integration.mp4| Very short video showing what this integration looks like when using the Watson Assistant preview panel.|
+|WA-IBM-RPA-Tester-action.json| Action skill sample that calls out to the RPA cloud functions using the above OpenAPI spec. |
+
+Note that the above templates will need to be modified according to the Integration Steps outlined below.
+
+## Integration Setup
+
+### Prerequisites
+
+To complete an end-to-end setup of this integration between Watson Assistant and IBM RPA you will need to provision accounts for all of the following:
+
+ - Watson Assistant (a trial account)
+ - IBM RPA
+ - IBM Cloud Functions (or some equivalent)
+
+### Setup RPA
+
+Go [here](https://www.ibm.com/docs/en/rpa/21.0?topic=automation-deploying-scripts) first for details on building and deploying RPA scripts.
+
+These instructions outline the basic steps: 
```

Comment:
consider adding images throughout the steps in the 3 Setup sections.

We are trying to make the instructions as easy as they can

Reply:
done